aws/amazon-ecs-ami
# Install aws-neuronx-tools in AL2INF and AL2023NEU

### Summary
This PR updates the `scripts/enable-ecs-agent-inferentia-support.sh` to install [aws-neuronx-tools](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/release-notes/tools/aws-neuronx-tools.html) for both __AL2INF__ and __AL2023NEU__. 

### Implementation details
Updates `scripts/enable-ecs-agent-inferentia-support.sh`.

### Testing
1. Build a custom ECS optimized AL2INF and AL2023NEU AMI with following parameters
```
ami_version          = "20990909"
ecs_agent_version    = "1.76.0"
ecs_init_rev         = "1"
docker_version       = "20.10.23"
docker_version_al2023 = "20.10.23"
containerd_version   = "1.6.19"
containerd_version_al2023 = "1.6.19"
source_ami_al2       = "amzn2-ami-minimal-hvm-2.0.20230926.0-x86_64-ebs"
source_ami_al2023    = "al2023-ami-minimal-2023.2.20230920.1-kernel-6.1-x86_64"
kernel_version_al2023    = "-kernel-6.1"
distribution_release_al2023  = "2023.2.20230920"
```
2. Run AMI tests on both custom AMIs, and confirm `NeuronDevices` test and `NeuronDevicesWithNeuronTools` test passed.
3. Run functional e2e tests and all tests passed on AL2INF and AL2023NEU. 

New tests cover the changes: no

### Description for the changelog
Install aws-neuronx-tools in AL2INF and AL2023NEU

### Licensing
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
